huxley/core/admin/country.py

```diff
@@ -17,10 +17,12 @@ class CountryAdmin(admin.ModelAdmin):
 
     def load(self, request):
         '''Import a CSV file containing countries.'''
+        # IMPORTANT: If the CSV file requirements change, make sure to update huxley/templates/admin/core/country/change_list.html
         countries = request.FILES
         reader = csv.reader(countries['csv'].read().decode('utf-8').splitlines())
         for row in reader:
-            special = False if row[1] == '0' or row[1] == 'False' or not row[1] else True
+            special = False if not row[1] or row[1].strip() == '0' or row[1].strip().lower() == 'false' or row[1].strip().lower() == 'n' else True
```

Comment:
seems to need the `.strip()` since space was included... Is this expected behavior? I thought CSVs were space agnostic.

Testfile used was:
```
testN, N
testn, n
testY, Y
testy, y
```